content/en/serverless/troubleshooting/serverless_tracing_and_webpack.md

```diff
@@ -2,15 +2,18 @@
 title: Node.js Lambda Tracing and Webpack Compatibility 
 kind: documentation
 further_reading:
-    - link: 'serverless/installation/node'
-      tag: 'Documentation'
+- link: '/serverless/installation/nodejs'
+  tag: 'Documentation'
+  text: 'Instrumenting Node.js Applications'
 ---
 
 # Compatibility
 
-Datadog's tracing libraries (`dd-trace`) are known to be not compatible with [webpack][1] due to the use of conditional imports and other issues. If using webpack and tracing your Node.js serverless functions:
+Datadog's tracing libraries (`dd-trace`) are known to be not compatible with [webpack][1] due to the use of conditional imports and other issues. While webpack cannot *build* `dd-trace`, your application can still *use* `dd-trace` and `datadog-lambda-js` provided by the *prebuilt* Datadog Lambda layer. Follow the instructions below:
 
-1. Mark `datadog-lambda-js` and `dd-trace` as [externals][2] for webpack, so webpack knows these dependencies will be available in the Lambda runtime.
+1. Follow the [installation instructions for Node.js][2] and ensure the Datadog Lambda layer is added to your Lambda function. 
+
+1. Mark `datadog-lambda-js` and `dd-trace` as [externals][3] for webpack, so webpack knows to skip building them as dependencies, because they are already available in the Lambda runtime provided by the Datadog Lambda layer.
```

Comment:
```suggestion
1. Follow the [installation instructions for Node.js][2] and ensure the Datadog Lambda layer is added to your Lambda function. 
2. Mark `datadog-lambda-js` and `dd-trace` as [externals][3] for webpack. This tells webpack to skip building them as dependencies, since they are already available in the Lambda runtime provided by the Datadog Lambda layer.
```